packages/documentation/src/stories/modules/header/language-switch/language-switch.stories.ts

```diff
@@ -4,7 +4,7 @@ import { MetaComponent } from '@root/types';
 
 const meta: MetaComponent<HTMLPostLanguageSwitchElement> = {
   id: 'decbb10c-2b39-4f47-b67d-337d8111a3ae',
-  title: 'Components/Header/Language Switch',
+  title: 'Modules/Header/Language Switch',
```

Comment:
The language switch is not a module:
```suggestion
  title: 'Components/Language Switch',
```

Reply:
I know, but I thought you want to have it at the same place, where the header docs are. At least this was the case before.